client/src/containers/projects/table/view/key-costs/index.tsx

```diff
@@ -64,10 +64,11 @@ export function KeyCostsTable() {
     {
       query: {
         ...filtersToQueryParams(filters),
-        fields: TABLE_COLUMNS.map((column) => column.accessorKey),
-        ...(sorting.length > 0 && {
-          sort: sorting.map((sort) => `${sort.desc ? "" : "-"}${sort.id}`),
-        }),
+        // fields: TABLE_COLUMNS.map((column) => column.accessorKey),
```

Comment:
After removing:
https://github.com/Vizzuality/tnc-blue-carbon-cost-tool/pull/77/files#r1832131548

TABLE_COLUMNS contains also properties inherited from BaseEntity, which are not part of the 'queryable' entity.

We should discuss how to manage this and we might be able to use the zod schema for projects. Picking or Omitting the props of BaseEntity might be hackish, maybe we can extract the available properties from the projectSchema? If so, ideally from the client itself

cc @andresgnlez